whenchanged/test_whenchanged.py

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env python
+
+import unittest
+try:
+    from whenchanged import whenchanged
+except:
+    import whenchanged
+
+class TestParseArgs(unittest.TestCase):
+    def assertSimilar(self, l1, l2):
+        return self.assertEqual(sorted(l1), sorted(l2))
+
+    def construct_WhenChanged(self, cmd):
+        return whenchanged.WhenChanged(**whenchanged.parse_args(cmd.split(' ')))
```

Comment:
I think it's better to test this without instantiating WhenChanged(). In other words, check that the returned dict from parse_args() contains the correct key/value pairs.